doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -277,3 +278,105 @@ Set used by the SPM sample
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52 DK <ug_nrf52>`                        |PCA10040     |:ref:`nrf52dk_nrf52832 <nrf52dk_nrf52832>`      |``nrf52dk_nrf52832``            |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+
+.. set21_end
+
+
+Set used by BLE sample (Peripheral GATT Discovery Manager)
+
+.. set22_start
+
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|Hardware platforms                                |PCA          |Board name                                      |Build target                    |
++==================================================+=============+================================================+================================+
+|:ref:`nRF52840 DK <ug_nrf52>`                     |PCA10056     |:ref:`nrf52840dk_nrf52840 <nrf52840dk_nrf52840>`|``nrf52840dk_nrf52840``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52840 DK (emulating nRF52811) <ug_nrf52>`|PCA10056     |:ref:`nrf52840dk_nrf52811 <nrf52840dk_nrf52811>`|``nrf52840dk_nrf52811``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52833 DK <ug_nrf52>`                     |PCA10100     |:ref:`nrf52833dk_nrf52833 <nrf52833dk_nrf52833>`|``nrf52833dk_nrf52833``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52833 DK (emulating nRF52820) <ug_nrf52>`|PCA10100     |:ref:`nrf52833dk_nrf52820 <nrf52833dk_nrf52820>`|``nrf52833dk_nrf52820``         |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52 DK <ug_nrf52>`                        |PCA10040     |:ref:`nrf52dk_nrf52832 <nrf52dk_nrf52832>`      |``nrf52dk_nrf52832``            |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+|:ref:`nRF52 DK (emulating nRF52810) <ug_nrf52>`   |PCA10040     |:ref:`nrf52dk_nrf52810 <nrf52dk_nrf52810>`      |``nrf52dk_nrf52810``            |
++--------------------------------------------------+-------------+------------------------------------------------+--------------------------------+
+
+.. set22_end
+
+Set used by BLE sample (Peripheral LBS)
```

Comment:
BLE -> Bluetooth LE

Also here a new group for a single sample?
Could we reduce the number of groups?

Reply:
New group as nRF52840 Dongle is also supported in this sample.